atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/persistentlock/KvsBackedPersistentLockService.java

```diff
@@ -50,7 +50,7 @@ public String releaseLock(LockEntry lockEntry) throws CheckAndSetException {
 
         try {
             lockStore.releaseLock(lockEntry);
-            return "The lock was released successfully.\n";
+            return "\"The lock was released successfully.\\n\"";
```

Comment:
We probably want some tests that verify the response conforms to expectations. Might be outside scope of this PR, but this stringly typed API seems unfortunate.